helm/hpcc/docs/placements.md

```diff
@@ -23,12 +22,55 @@ The list item in "pods" can be one of the following:
 
 Supported configurations under each "placement"
 1) nodeSelector
+   Multiple nodeSelectors can be applied. For example
+   placements:
+   - pods ["all"]
```

Comment:
missing colon (and in other places below for 'pods')

Reply:
fixed